thinc/config.py

```diff
@@ -142,6 +142,7 @@ class registry(object):
     layers: Decorator = catalogue.create("thinc", "layers", entry_points=True)
     losses: Decorator = catalogue.create("thinc", "losses", entry_points=True)
     initializers: Decorator = catalogue.create("thinc", "initializers", entry_points=True)
+    keras: Decorator = catalogue.create("thinc", "keras", entry_points=True)
```

Comment:
This should probably live in the wrapper module file and not in the global `registry`, since it's specific to the wrapper.